core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -45,10 +45,17 @@ const (
 
 // block is used to key into a set that tracks beacon blocks.
 type block struct {
+	hash        common.Hash
 	blockNumber uint64
 	confDelay   uint32
 }
 
+type callback struct {
+	hash        common.Hash
```

Comment:
Should we call this `blockHash` to make it clear that this hash refers to a block hash?

Reply:
i can do so, Block just leaves it as hash but we can do better on our end.